dialogue-core/src/main/java/com/palantir/dialogue/core/RetryingChannel.java

```diff
@@ -433,6 +445,48 @@ private String channelName() {
         }
     }
 
+    private static final class ConsumptionTrackingRequestBody implements RequestBody {
```

Comment:
We're missing a delegating override on `contentLength()`

Reply:
🤦‍♂️ Thank you